README.md

```diff
@@ -76,11 +98,13 @@ For more details about the output files and reports, please refer to the
 
 ## Credits
 
-nf-core/riboseq was originally written by Caroline.
-
-We thank the following people for their extensive assistance in the development of this pipeline:
+nf-core/riboseq was originally written by [Jonathan Manning](https://github.com/pinin4fjords) (Bioinformatics Engineer as Seqera) with support from [Altos labs](https://www.altoslabs.com/) and in discussion with [Felix Krueger](https://github.com/FelixKrueger) and [Christel Krueger](https://github.com/ChristelKrueger). We thank the following people for their input:
```

Comment:
Bioinformatics Engineer _at_ Seqera?

Reply:
Thanks, addressed in https://github.com/nf-core/riboseq/pull/50